.circleci/config.yml

```diff
@@ -715,20 +717,18 @@ jobs:
                 pattern: '^pull/.+$'
                 value: << pipeline.git.branch >>
           steps:
+            - aws-cli/setup:
```

Comment:
Usually we try to run by the principle of least privilege. Scoping down secrets to the individual step if it makes sense.

Reply:
CI failed without having it as a separate step - https://app.circleci.com/pipelines/github/mui/material-ui/106485. My initial commit was only https://github.com/mui/material-ui/pull/38566/commits/7b0da0d638312d585cf7246e88446117bd7d717e. So looks like `aws-cli` is required.